fusillade/directory/structs.py

```diff
@@ -0,0 +1,38 @@
+from collections import namedtuple
+from enum import Enum, auto
+
+
+class UpdateActions(Enum):
+    CREATE_OR_UPDATE = auto()
+    DELETE = auto()
+
+
+obj_type_path = dict(
+    group='/group/',
+    index='/index/',
+    user='/user/',
+    policy='/policy/',
+    role='/role/',
+    resource='/resource/'
+)
+
+
+class ValueTypes(Enum):
+    StringValue = auto()
+    BinaryValue = auto()
+    BooleanValue = auto()
```

Comment:
Just out of curiosity, what's the difference between `BinaryValue` and `BooleanValue`?

Reply:
**BinaryValue**
A binary data value.
Type: Base64-encoded binary data object

**BooleanValue**
A Boolean data value.
Type: Boolean

https://docs.aws.amazon.com/clouddirectory/latest/APIReference/API_TypedAttributeValue.html